ios/FluentUI/Card/CardView.swift

```diff
@@ -72,24 +72,6 @@ public enum CardSize: Int, CaseIterable {
         }
     }
 
-    var primaryTextStyle: TextStyle {
```

Comment:
With CardView no longer switching style of text based on small vs large, are we still in line with design in these two cases? I assume yes but wanted to double check.

Reply:
i looked at the design figma and it didn't have the larget case for card. it only had small cases. but even the real file preview had the same font size so i decided to simplify it.